### PR TITLE
Boilerplate fix

### PR DESCRIPTION
**Pull request by d4v1d41**

### CODE REVIEW

File: frontend/src/pages/home.pages.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { FC } from "react";
+
+import { RouteComponentProps } from "@reach/router";
+
+type TParams = { id: string };
+
+export const HomePage: FC<RouteComponentProps> = (props: RouteComponentProps<TParams>) => {
+  console.log(props)
```
Comment (d4v1d41) [inline frontend/src/pages/home.pages.tsx]: There is a console log here. But most importantly, let's keep the Dashboard page only for now.

File: frontend/src/routes.tsx [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 import { Layout } from './layout/layout.component';
 import { DashboardPage } from './pages/dashboard.page';
+import { HomePage } from "./pages/home.pages";
 
 export function Routes() {
   return (
     <Layout>
       <Router>
-        <DashboardPage/>
+        <HomePage path="/home"/>
```
Comment (d4v1d41) [inline frontend/src/routes.tsx]: Let's add the default prop to the route:

image


Please, take a look at the documentation for Reach Router: https://reach.tech/router/tutorial/08-default-routes

File: frontend/src/pages/dashboard.page.tsx [hunk 3]
```diff
@@ -2,4 +2,10 @@ import React, { FC } from "react";
 
 import { RouteComponentProps } from "@reach/router";
 
-export const DashboardPage: FC<RouteComponentProps> = () => {};
+type TParams = { id: string };
+
+export const DashboardPage: FC<RouteComponentProps> = (props: RouteComponentProps<TParams>) => {
+  return (
+    <div style={{ margin: "5rem", textAlign: "center" }}>Dashboard</div>
```
Comment (d4v1d41) [inline frontend/src/pages/dashboard.page.tsx]: Here you just added content to the page. Let's move to done the corresponding task on Trello.

File: frontend/src/layout/navbar.tsx [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
+import { RouteComponentProps } from "@reach/router";
 
 export const NavBar: React.FC<RouteComponentProps> = () => {
   return (
     <AppBar position="fixed">
+      <div style={{ display: "flex", justifyContent: "center" }}>
       <Button color="inherit" href="/dashboard">
         People's Dashboard
       </Button>
+      <Button color="inherit" href="/home">
```
Comment (d4v1d41) [inline frontend/src/layout/navbar.tsx]: Let's remove any additional code not specified by the tasks. We must maintain our solution as clean as possible.

File: frontend/src/interfaces/person.dto.ts [hunk 5]
```diff
@@ -1,4 +1,4 @@
-export interface IPersonDTO {
+export interface IPersonProps {
```
Comment (d4v1d41) [inline frontend/src/interfaces/person.dto.ts:1]: Could you please justify why did you rename the interface to IPersonProps?

File: frontend/.gitignore [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+.rpt2_cache/
+.rts2_cache_cjs/
+.rts2_cache_es/
+.rts2_cache_umd/
+
+# Optional REPL history
+.node_repl_history
+
+# Output of 'npm pack'
+*.tgz
```
Comment (d4v1d41) [inline frontend/.gitignore]: Do we really need these in our gitignore? Please validate that what you've added is fine for our solution, we might be adding additional ignores that we are not currently using.

File: frontend/src/pages/dashboard.page.tsx (hunk 3 above)
Comment (814942) [reply frontend/src/pages/dashboard.page.tsx]: I need to add content here to fix the error that is showing up

File: frontend/src/interfaces/person.dto.ts (hunk 5 above)
Comment (814942) [reply frontend/src/interfaces/person.dto.ts:1]: Because the name does not match the import that is used in person-card. I think it is better to change the name of the interface instead of the import name and the property name

File: frontend/.gitignore (hunk 6 above)
Comment (814942) [reply frontend/.gitignore]: Yes, I believe some of these cases don't match, but it's a standard file I use to ignore most of the files I don't want to load.

File: frontend/src/pages/home.pages.tsx (hunk 1 above)
Comment (d4v1d41) [reply frontend/src/pages/home.pages.tsx]: It looks like the home page is still here.

File: frontend/.gitignore (hunk 6 above)
Comment (d4v1d41) [reply frontend/.gitignore]: By adding ignores to dependencies/folder/files we don't have, we are adding lines that do not belong to this codebase. Let's remove them all. Instead we should only have:
- .env
- dist
- node_modules

File: frontend/src/components/notFound/not-found.css [hunk 7]
```diff
@@ -0,0 +1,16 @@
+.container_not_found {
```
Comment (d4v1d41) [inline frontend/src/components/notFound/not-found.css]: I understand your point for this, but this is out of scope for now. Let's remove it too. The whole not found folder, for now.

Comment (d4v1d41) [reply frontend/src/components/notFound/not-found.css]: As well as the asset you uploaded (not found image).
